src/main/resources/application.yml

```diff
@@ -7,6 +7,10 @@ server:
     context-path: /api
 
 spring:
+  mvc:
+    pathmatch:
+      matching-strategy: ant_path_matcher
+
```

Comment:
이 설정이 과연.. Swagger 설정하는것과 연관성이 있을까여 ..?

Reply:
엇 근데 저거 설정을 안해주면
org.springframework.context.ApplicationContextException: Failed to start bean 'documentationPluginsBootstrapper'; nested exception is java.lang.NullPointerException: Cannot invoke "org.springframework.web.servlet.mvc.condition.PatternsRequestCondition.toString()" because the return value of "springfox.documentation.spi.service.contexts.Orderings.patternsCondition(springfox.documentation.RequestHandler)" is null
이 에러가 뜹니당..

Reply:
https://u-comensee.tistory.com/165
이렇다고 하여서 추가했는데.. 다르게 할수있는 방법이 있나욥..